src/main/utility/interval_map.rs

```diff
@@ -99,6 +99,15 @@ impl<V: Clone> IntervalMap<V> {
         ItemIter { map: self, i: 0 }
     }
 
+    pub fn iter_from(&self, begin: usize) -> ItemIter<V> {
```

Comment:
Maybe add a comment for this function since all of the other functions are well documented?

Reply:
Done